packages/terra-consumer-nav/src/components/nav-items/NavItem.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import styles from './NavItem.scss';
+import NavToggler from '../nav-toggler/NavToggler';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  // do we need these...
+  slug: PropTypes.string,
+  navType: PropTypes.oneOf(['GROUPING', 'EXTERNAL', 'APPLICATION', 'MODAL']),
+  target: PropTypes.string,
+  //
+  uri: PropTypes.string.isRequired,
+  text: PropTypes.string.isRequired,
+  isActive: PropTypes.bool,
+
+  // only those that need toggler
+  isSubNav: PropTypes.bool,
```

Comment:
no, since not all higher level level nav items have subnavs, checking on that would make those render as sub-navs